modules/bmo-configuring-userdata.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-userdata_{context}"]
+= Configuring userData
+
+The cloud image for most operating systems provide the `cloud-init` service, which can read custom scripts you create in `userData` format.
+
+The `userData` field in the `BareMetalHost` custom resource allows you to customize the bare-metal instance during provisioning by providing initial configuration data, such as user credentials or system settings. For most operating systems this is achieved using a `cloud-init` script encapsulated in a Kubernetes secret. 
+
+[NOTE]
+====
+For {op-system-first}, use Ignition.
```

Comment:
this comes out as 
"For Red Hat Enterprise Linux CoreOS (RHCOS), use Ignition."
but we don't support RHCOS on BMaaS (only Fedora/CentOS CoreOS)
let's revisit and make sure this is expanded in a way that is consistent with other OS references.

Reply:
Yeah. They have some conditions in the attributes file, so I just inlined the text.